T-1/git-backup-helper/setup-aliases.sh

```
@@ -0,0 +1,52 @@
+add_alias "gitbackup" "${SCRIPT_PATH}"
+add_alias "gitfullbackup" "${SCRIPT_PATH} --full"
+
+# 設置執行權限
+chmod +x "${SCRIPT_PATH}"
+
+# 輸出提示信息
+echo -e "\n${GREEN}別名設置完成!${NC}"
+echo "請運行以下命令使變更生效:"
+echo "source ${CONFIG_FILE}"
+echo -e "\n使用方式:"
+"echo "  1. 在 Git 倉庫目錄中執行: gitbackup"
```

Shellcheck (0.10.0)

[warning] This is interpreted as a command name ending with space. Double check syntax.

(SC2288)